guides/common/attributes-base.adoc

```diff
@@ -17,11 +17,11 @@
 :RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
 :RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
 // For Beta, change to "Beta". For GA releases, change to, for example, "6.8".
-:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-6.9-rpms
-:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-7-server-satellite-6.8-rpms
-:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-6.9-rpms
-:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6.9-rpms
-:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-6-rpms
+:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-6-beta-rpms
```

Comment:
Can't we introduce a macro for the `6-beta` part that we later flip to `{ProductVersion}`?

Reply:
@ekohl I'm not sufficiently comfortable doing this myself - can I create an issue for this enhancement for GA and prioritize merging this as-is? thanks

Reply:
It's challenging release for docs team, let's just do it the regular way, shall we?